Cabal-tests/tests/UnitTests/Distribution/Utils/Json.hs

```diff
@@ -0,0 +1,42 @@
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE OverloadedStrings #-}
+module UnitTests.Distribution.Utils.Json
+    ( tests
+    ) where
+
+import Distribution.Utils.Json
+
+import Test.Tasty
+import Test.Tasty.HUnit
+
+tests :: [TestTree]
+tests =
+    [ testCase "escapes strings correctly" $
+      renderJson (JsonString "foo\"bar") @?= "\"foo\\\"bar\""
+    , testCase "renders empty list" $
+      renderJson (JsonArray []) @?= "[]"
+    , testCase "renders singleton list" $
+      renderJson (JsonArray [JsonString "foo\"bar"]) @?= "[\"foo\\\"bar\"]"
+    , testCase "renders list" $
+      renderJson (JsonArray [JsonString "foo\"bar", JsonString "baz"]) @?= "[\"foo\\\"bar\",\"baz\"]"
+    , testCase "renders empty object" $
+      renderJson (JsonObject []) @?= "{}"
+    , testCase "renders singleton object" $
+      renderJson (JsonObject [("key", JsonString "foo\"bar")]) @?= "{\"key\":\"foo\\\"bar\"}"
+    , testCase "renders object" $
+      renderJson (JsonObject
+                    [ ("key", JsonString "foo\"bar")
+                    , ("key2", JsonString "baz")])
+          @?= "{\"key\":\"foo\\\"bar\",\"key2\":\"baz\"}"
```

Comment:
@Mikolaj, @emilypi Do we have some form of formatting standards for new code?

Reply:
There's been no concrete discussion of standards. Don't worry about it for now, and we'll come up with one

Reply:
Right. We only have https://github.com/haskell/cabal/blob/master/CONTRIBUTING.md#conventions, as mentioned in PR template. I guess, for now, use whatever conventions exist in the snippet you edit, unless they are utterly insane (in which case, create a reformatting commit first).